packages/nextjs-contentful-renderer/src/getAffectedSlugs/getAffectedSlugs.ts

```diff
@@ -0,0 +1,78 @@
+import { getBlocksByModelIds } from 'src/getBlocksByModelIds';
+import { getPagesByBlockIds } from 'src/getPagesByBlockIds';
+import { getReplicaById } from 'src/getReplicaById';
+import { getReplicasByPageIds } from 'src/getReplicasByPageIds';
+import { getRouteById } from 'src/getRouteById';
+import { getRoutesByVariantIds } from 'src/getRoutesByVariantIds';
+import { getVariantsByPageIds } from 'src/getVariantsByPageIds';
+import { ContentType, ContentfulWebhookPayload } from './types';
+
+export async function getAffectedSlugs(
+  webhookPayload: ContentfulWebhookPayload,
```

Comment:
Should this method expect the `webhookPayload` or should we expect just the `slug`, `contentType` and `entryId`?